src/async/gateway.rs

```diff
@@ -41,17 +39,17 @@ impl Gateway {
         header: &str,
         body: &str,
         ok: &str,
-    ) -> Box<Future<Item = RequestReponse, Error = RequestError>> {
+    ) -> Box<Future<Item = RequestReponse, Error = RequestError> + Send> {
```

Comment:
Why is `Send` required everywhere now?

Reply:
if you don't declare everything like this you can't use it with the tokio executors... it's just, a thing, afaik.